pkg/chaosdaemon/pb/chaosdaemon.proto

```diff
@@ -233,6 +233,8 @@ message ApplyHttpChaosRequest {
   int64 startTime = 5;
   bool enterNS = 6;
   string instance_uid = 7;
+
+  string tls = 8;
```

Comment:
IMHO, using JSON string inside of a protocol is definitely anti-pattern (like multi-value in one column in RDBMS)

it means we are using convention instead of schema/protocol, that's ambiguous.

Reply:
I think this maybe the anti-pattern of protocol like json. But not the anti-pattern of RPC, especially we use it just like running a span of code in two machine. The main problem is gRPC cannot do tls parsing and unparsing for us.

Reply:
> The main problem is gRPC cannot do tls parsing and unparsing for us.

It seems that the content of `tls` is `[]PodHttpChaosBaseRule`, which is complex but it does not mean there is no way to describe it in protobuf. 

But I think it's too heavy to make this PR carry these changes, so I would not block this PR even this does not look good to me.

Similar issues also happen on stress chaos and jvm chaos, we should refine the protobuf file before it becomes too hard to maintance.